tests/swe_radnorm.phpt

```diff
@@ -0,0 +1,19 @@
+--TEST--
+Basic test
+--SKIPIF--
+<?php
+if (!extension_loaded('swephp')) {
+    echo 'skip';
+}
+?>
+--FILE--
+<?php
+swe_set_ephe_path('./sweph/ephe');
+var_dump(swe_radnorm(0));
+var_dump(swe_radnorm(6.2831853071796));
+var_dump(round(swe_radnorm(-0.017453292519943), 6));
```

Comment:
I'm a little concerned about using such high-precision due to minor architecture differences across systems when running tests.

Reply:
so far, we have no had problems caused by precision. But we work both on Intel 64bit.